### PR TITLE
STORM-2251 Integration test refers specific version of Storm which should be project version

### PR DESCRIPTION
**Pull request by HeartSaVioR**
* fix integration-test's pom.xml to have root pom as parent, and remove version and storm version

### CODE REVIEW

File: integration-test/pom.xml [hunk 1]
```diff
@@ -48,33 +53,6 @@
...
-        </repository>
-        <repository>
-            <id>hortonworks</id>
-            <url>http://nexus-private.hortonworks.com/nexus/content/groups/public/</url>
-        </repository>
-    </repositories>
-
     <dependencies>
         <dependency>
             <groupId>org.testng</groupId>
```
Comment (srdo) [inline integration-test/pom.xml]: Consider removing joda-time version too, it's defined in the root pom's dependency management.

**Comment by srdo** [comment]
+1

File: integration-test/pom.xml (hunk 1 above)
Comment (HeartSaVioR) [inline integration-test/pom.xml]: Good point. Will address.

Comment (HeartSaVioR) [inline integration-test/pom.xml]: Addressed.

**Comment by harshach** [comment]
+1

**Pull request merged**